src/openrct2/paint/tile_element/Surface.cpp

```diff
@@ -666,13 +666,12 @@ static void viewport_surface_draw_tile_side_bottom(paint_session * session, enum
         if (curHeight != tunnelArray[0].height)
```

Comment:
Missing `tunnelIndex`?

Reply:
Oh yeah

Reply:
It still does it even with tunnelIndex there, there is something I must be missing.

Reply:
Got it, stupid me.